packages/functional-tests/pages/signinTotpCode.ts

```diff
@@ -0,0 +1,14 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { BaseTokenCodePage } from './baseTokenCode';
+
+export class SigninTotpCodePage extends BaseTokenCodePage {
+  readonly path = '/signin_totp_code';
+
+  get input() {
+    this.checkForPath(this.path);
+    return this.page.locator('input[type=number]');
```

Comment:
Here we override the default functionality of the base class, since the totp code has a slightly different selector.

Reply:
I believe in this case a user facing attribute would make this override unnecessary. Can we clean this up, when update locators?  If so, at that point only the baseTokenCodePage will be needed, and we can just pass the expected path into the constructor as you suggest. I'd be 100% okay with that approach too.